webapp/src/test/java/com/box/l10n/mojito/monitoring/QuartzPendingJobsReportingTaskTest.java

```diff
@@ -0,0 +1,143 @@
+package com.box.l10n.mojito.monitoring;
+
+import com.box.l10n.mojito.Application;
+
+import org.awaitility.core.ConditionFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.quartz.Job;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.Scheduler;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.security.SecureRandom;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.search.MeterNotFoundException;
+import io.micrometer.core.instrument.search.RequiredSearch;
+
+import static com.box.l10n.mojito.quartz.QuartzConfig.DYNAMIC_GROUP_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.quartz.JobBuilder.newJob;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {Application.class},
+                properties = {
+                    "l10n.management.metrics.quartz-metrics.enabled=true",
+                    "l10n.management.metrics.quartz-metrics.execution-rate=PT1S" })
+public class QuartzPendingJobsReportingTaskTest {
+
+    public static final String MY_GROUP = "MY_GROUP";
+
+    @Autowired
+    MeterRegistry meterRegistry;
+
+    @Autowired
+    Scheduler scheduler;
+
+    ConditionFactory await;
+    SecureRandom random = new SecureRandom();
+
+    @Before
+    public void setUp() throws Exception {
+        if (!scheduler.isStarted()){
+            scheduler.start();
```

Comment:
in what case would this happen?

Reply:
I had a couple of failed situations when running my tests locally, so to avoid any failures it was better to have this as part of the setup.